wagtail/models/audit_log.py

```diff
@@ -82,7 +82,7 @@ def with_instances(self):
             else:
                 # The model class for the logged instance no longer exists,
                 # so we have no instance to return. Return None instead.
-                model_instances = {object_id: None for object_id in object_ids}
+                model_instances = dict.fromkeys(object_ids)
```

Comment:
Is this change needed for this fix? If not, maybe revert it and feel free to raise a seperate PR for this if it's a worthwhile improvement.